File: src/ois_andibandi/andibandi.c
// NOTE: it is recommended to use this even if you don't understand the following code.

#include <assert.h>
#include <stdio.h>
#include <stdlib.h>
#include <stdbool.h>

int D, Sa, Sb, Ta, Tb, Ua, Ub;

int main() {
    // uncomment the two following lines if you want to read/write from files
    // freopen("input.txt", "r", stdin);
    // freopen("output.txt", "w", stdout);

    assert(3 == scanf("%d%d%d", &Sa, &Ta, &Ua));

    assert(3 == scanf("%d%d%d", &Sb, &Tb, &Ub));

    D = 0;


    // INSERT YOUR CODE HERE


    printf("%d\n", D);

    return 0;
}

File: src/ois_andibandi/andibandi.java
// NOTE: it is recommended to use this even if you don't understand the following code.

import java.util.*;
import java.io.*;
import java.lang.*;

public class andibandi {

    public static void main(String[] args) throws FileNotFoundException, IOException {
        Locale.setDefault(Locale.US);
        InputStream fin = System.in;
        OutputStream fout = System.out;
        // uncomment the two following lines if you want to read/write from files
        // fin = new FileInputStream("input.txt");
        // fout = new FileOutputStream("output.txt");

        BufferedWriter writer = new BufferedWriter(new OutputStreamWriter(fout));
        reader = new BufferedReader(new InputStreamReader(fin));
        scn = new StringTokenizer(reader.readLine());

        int Sa = Integer.parseInt(next());
        int Ta = Integer.parseInt(next());
        int Ua = Integer.parseInt(next());

        int Sb = Integer.parseInt(next());
        int Tb = Integer.parseInt(next());
        int Ub = Integer.parseInt(next());

        int D = 0;


        // INSERT YOUR CODE HERE


        writer.write(String.valueOf(D));
        writer.write('\n');

        writer.flush();
    }

    static String next() throws IOException {
        while (!scn.hasMoreTokens()) scn = new StringTokenizer(reader.readLine());
        return scn.nextToken();
    }

    static BufferedReader reader;
    static StringTokenizer scn;
}
